app/api/gltf/gltf.controller.js

```diff
@@ -91,7 +103,20 @@ async function getDbImage(image_id) {
   return result;
 }
 
-async function createGltfFromImageCoordinates(imageCoordinates, image_id, collection_id) {
+async function createGltfFromImageCoordinates(imageCoordinates, image_id, collection_id, regionByPx) {
+  const imageSquaredFromDB = await getDbImage(image_id);
+
+  let region_url = "";
+  let picPath;
+  if (regionByPx) {
+    //name of the gltf file
+    region_url = `_${regionByPx[0]}_${regionByPx[1]}_${regionByPx[2]}_${regionByPx[3]}`;
+    //path to the iiif image in the gltf file
+    const iiif_base_url = imageSquaredFromDB.media.tiles.url.replace('info.json','');
+    picPath = `${iiif_base_url}${regionByPx[0]},${regionByPx[1]},${regionByPx[2]},${regionByPx[3]}/max/0/default.jpg`;
```

Comment:
move to get `getDbImage`

- add extra optional parameter `regionByPX`
- if parameter given use it, else use regionByPx crop from DB